webrender/src/util.rs

```diff
@@ -508,3 +508,67 @@ pub fn world_rect_to_device_pixels(
     let device_rect = rect * device_pixel_scale;
     device_rect.round_out()
 }
+
+/// Run a first callback over all elements in the array. If the callback returns true,
+/// The element is removed from the array and moved to a second callback.
+///
+/// This is a simple implementation waiting for Vec::drain_filter to be stable.
+/// When that happens, code like:
+///
+/// ```
+/// let filter = |&mut op| {
```

Comment:
can we have that in unit tests instead of (or in addition to) the documentation?

Reply:
I don't think you can even write `|&mut op|` so checking these by the compiler would help ;)